src/System.IO.FileSystem/tests/FileInfo/GetSetTimes.cs

```diff
@@ -18,6 +18,8 @@ public override FileInfo GetExistingItem()
             return new FileInfo(path);
         }
 
+        private static bool HasNonZeroNanoSeconds(DateTime dt) => dt.Ticks % 10 != 0;
+        
```

Comment:
For consistency can you name this `GetNonZeroMilliseconds` ?

Reply:
but we are checking for nanoseconds here ?

Reply:
I meant the existing function below :)